crowbar_framework/app/models/service_object.rb

```diff
@@ -1543,9 +1561,10 @@ def run_remote_chef_client(node, command, logfile_name)
 
   private
 
-  def wait_for_chef_clients(node)
-    unless RemoteNode.chef_ready?(node, 1200)
-      STDERR.puts "Waiting for already running chef-clients on #{node} failed"
+  def wait_for_chef_clients(node_name)
+    @logger.debug("wait_for_chef_clients: Waiting for already running chef-clients on #{node_name}.")
```

Comment:
> Hrm, does this work fine from the other place where this is being called (run_remote_chef_client)? (it's from a fork, so I'm not sure @logger.debug worls there)

Good point, I have no clue I would need to test that. You have any suggestions?

> Actually, with this change, we can probably remove that other wait_for_chef_clients call, can't we?

Not sure if I understand. which other `wait_for_chef_clients` call are you referring to?